### PR TITLE
updated refs and links

### PR DESCRIPTION
**Pull request by kpathakota**
Hope this is what you meant @kevinslin. I didn't see anything else to add in the release notes.

### CODE REVIEW

File: vault/dendron.topic.links.md [hunk 1]
```diff
@@ -19,6 +19,7 @@ Dendron supports multiple types of links and formats.
 ## Wiki Links
 Wiki links support **autocomplete**. To initiate:
 - create wikilink brackets and start typing
+- link to specific sections of notes using the `[[wiki link#header]]` syntax
```
Comment (kevinslin) [inline vault/dendron.topic.links.md:22]: currently relative wiki links will now work in preview or when exported as a pod but they will work in publishing. can we add those caveats?

File: vault/dendron.topic.refs.md [hunk 2]
```diff
@@ -54,6 +54,12 @@ A note reference will include the entire contents of a note within another note.
 
 ![](https://foundation-prod-assetspublic53c57cce-8cpvgjldwysl.s3-us-west-2.amazonaws.com/assets/images/refs.block.gif)
 
+Note that references also respect the wildcard operator. The following example will include all child pages of sample.
```
Comment (kevinslin) [inline vault/dendron.topic.refs.md:57]: can we add this as its own section? eg. `Block Reference with Wildcard`

Comment (kpathakota) [reply vault/dendron.topic.refs.md:57]: I thought about this but isn't the wildcard operator usable in the three other forms of using refs? I.e. you can use the wildcard operator in note refs, block refs and block range refs. I didn't think this is actually a 4th "type".

File: vault/dendron.topic.links.md (hunk 1 above)
Comment (kpathakota) [reply vault/dendron.topic.links.md:22]: Yep, will do.

File: vault/dendron.topic.refs.md (hunk 2 above)
Comment (kevinslin) [reply vault/dendron.topic.refs.md:57]: that's fair

**Pull request merged**